### DISCUSSION TITLE
PPStructure结果中没有excel文件

### DISCUSSION THREAD

**Discussion by JasonChenJC** [PaddlePaddle/PaddleOCR#12622] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：MAC OS 14
- 版本号/Version：Paddle：  2.3.2 PaddleOCR： 2.7.0.3 问题相关组件/Related components：
- 运行指令/Command Code：

table_engine = PPStructure(show_log=True)
save_folder = './output'
img_path = 'xx.jpg'
img = cv2.imread(img_path)
result = table_engine(img)
save_structure_res(result, save_folder,os.path.basename(img_path).split('.')[0])

for line in result:
    line.pop('img')
    print(line)

from PIL import Image

font_path = './simfang.ttf' # PaddleOCR下提供字体包
image = Image.open(img_path).convert('RGB')
im_show = draw_structure_result(image, result,font_path=font_path)
im_show = Image.fromarray(im_show)
im_show.save('./result.jpg')

- 完整报错/Complete Error Message：
输出结果中只有jpg和txt，没有excel

**Comment by harvinyou** [comment]
这个是识别不出来了，我也有遇到这个问题

**Comment by harvinyou** [comment]
调整一下阈值
